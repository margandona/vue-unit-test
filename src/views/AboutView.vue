<template>
    <div class="about">
      <h1>This is the About page</h1>
    </div>
  </template>
  
  <script>
  export default {
    name: 'AboutView'
  }
  </script>
  
  <style scoped>
  .about {
    text-align: center;
  }
  </style>
  