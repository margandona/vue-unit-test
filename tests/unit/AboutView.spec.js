import { shallowMount } from '@vue/test-utils'
import AboutView from '@/views/AboutView.vue'

describe('AboutView.vue', () => {
  it('renders the About page', () => {
    const wrapper = shallowMount(AboutView)
    expect(wrapper.text()).toMatch('This is the About page')
  })

  it('matches snapshot', () => {
    const wrapper = shallowMount(AboutView)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
