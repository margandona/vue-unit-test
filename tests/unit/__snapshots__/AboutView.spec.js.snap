// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutView.vue matches snapshot 1`] = `
<div class="about">
  <h1>This is the About page</h1>
</div>
`;
